crest/Assets/Crest/Crest/Scripts/LodData/Settings/SimSettingsWave.cs

```diff
@@ -9,16 +9,19 @@ namespace Crest
     [CreateAssetMenu(fileName = "SimSettingsDynamicWaves", menuName = "Crest/Dynamic Wave Sim Settings", order = 10000)]
     public class SimSettingsWave : SimSettingsBase
     {
-        [Header("Simulation Settings")]
-        [Range(0f,32f), Tooltip("The wave sim will not run if the simulation grid is smaller in resolution than this size. Useful to limit sim range for performance.")]
+        [Header("Range")]
+        [Range(0f,32f), Tooltip("NOT CURRENTLY WORKING. The wave sim will not run if the simulation grid is smaller in resolution than this size. Useful to limit sim range for performance.")]
         public float _minGridSize = 0f;
-        [Range(0f, 32f), Tooltip("The wave sim will not run if the simulation grid is bigger in resolution than this size. Zero means no constraint/unlimited resolutions. Useful to limit sim range for performance.")]
+        [Range(0f, 32f), Tooltip("NOT CURRENTLY WORKING. The wave sim will not run if the simulation grid is bigger in resolution than this size. Zero means no constraint/unlimited resolutions. Useful to limit sim range for performance.")]
```

Comment:
What do you mean but this not currently working, should we remove these then?

Reply:
These seem to have broken at some indiscriminate time in the past. I swear they worked once. I thought the move to TAs might have broken them but it seems they were broken before that. I could comment them out, but if anyone has settings for these it will kill their data, so was going to leave as so. Let me know what you think.

Reply:
Shall we create an issue for this once this is merged?

Reply:
yeah, maybe i'll get your help with this one if you're up for it?